src/shell.ts

```diff
@@ -15,7 +17,12 @@ function createWindow() {
   mainWindow.once('ready-to-show', () => mainWindow.show());
 }
 
-app.whenReady().then(() => {
+app.whenReady().then(async () => {
+  const name = await installExtension(ANGULAR_DEVTOOLS)
+  if (!!name) {
+    console.log('An error occurred when downloading the extension: ', name);
+  }
+  console.log(`Added Extension:  ${name}`)
```

Comment:
Can we extract this block of code into a separate function?

Reply:
I've made a function for this as requested.